nucypher/utilities/porter/porter.py

```diff
@@ -14,12 +14,30 @@
  You should have received a copy of the GNU Affero General Public License
  along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
-from constant_sorrow.constants import NO_CONTROL_PROTOCOL
```

Comment:
Porter living inside a `utilities` module is short-selling it's greatness, and a case in favor of moving to a separate repo IMO (while I still do greatly appreciate the benefits of a unified CI).

Reply:
@derekpierre and I talked about this a while ago and we agreed that a separate repo is indeed a potential outcome. We also discussed the difficulties that we've seen with nucypher-monitor living in a separate repo in terms of synchronization. I'd personally lean towards keeping it inside the same repo, potentially relocated to a more prominent namespace.